src/Umbraco.Web.UI.Client/src/views/propertyeditors/templatablegrid/template.prevalues.controller.js

```diff
@@ -0,0 +1,75 @@
+﻿(function () {
+
+    function findScopeValue(scope, key) {
+        if (!scope) {
+            return null;
+        }
+        if (scope[key]) {
+            return scope[key];
+        }
+        if (scope.$parent) {
+            return findScopeValue(scope.$parent, key);
+        }
+        return null;
+    }
+
+    function findPreValue(array, alias) {
+        if (!array) {
+            return null;
+        }
+        return $.grep(array, function (item) {
+            return item.alias === alias;
+        })[0];
+    }
+
+    function TemplatableGridPrevaluesController(scope, win) {
+        var preValues = findScopeValue(scope, "preValues"),
+            layoutConfig = findPreValue(preValues, "items"),
```

Comment:
I don't think editorState will give me the right entity here, so I didn't bother trying.
It also fails in "dialog mode" in the document type editor. Hence setting "unsupported" on line 31. See view.